Makefile

```diff
@@ -69,26 +69,42 @@ test-all-components-coverage: ## Run tests with coverage
 		$(MAKE) test-components-coverage GROUP=$$(basename $$component_group) ; \
 	done
 
-sync-assets: ## Sync assets folder to GCS.
-	@if [ -d "./pipelines/assets/" ] ; then \
-		echo "Syncing assets to GCS" && \
-		gsutil -m rsync -r -d ./pipelines/assets ${PIPELINE_FILES_GCS_PATH}/assets ; \
-	else \
-		echo "No assets folder found" ; \
-	fi ;
-
-run: ## Compile pipeline, copy assets to GCS, and run pipeline in sandbox environment. Must specify pipeline=<training|prediction>. Optionally specify enable_pipeline_caching=<true|false> (defaults to default Vertex caching behaviour)
+build-packages: ## Build Python packages. Remember to deactivate your current virtual environment
```

Comment:
Might be worth having a Make command to check whether current shell is an activated virtualenv using the `$VIRTUAL_ENV` variable? This issue relates to the other commands like `compile-pipeline` and `run` too.